src/comm/bench-flow.js

```diff
@@ -170,11 +171,6 @@ function processResult(results, label){
             resultTable[1] = getResultValue(r);
         }
 
-        let sTP = r.sTPTotal / r.length;
-        let sT = r.sTTotal / r.length;
-        logger.debug('sendTransactionProposal: ' + sTP + 'ms length: ' + r.length);
-        logger.debug('sendTransaction: ' + sT + 'ms');
-        logger.debug('invokeLantency: ' + r.invokeTotal / r.length + 'ms');
```

Comment:
These results are needed for testing.

Reply:
Re-added, not sure why I removed these...